src/core.c/IO/Path.pm6

```diff
@@ -490,6 +490,27 @@ my class IO::Path is Cool does IO {
         }}
         nqp::chmod($.absolute, nqp::unbox_i($mode));
     }
+    method chown(IO::Path:D: :$uid is copy, :$gid is copy --> True) {
+
+        my str $path = self.absolute;
+        $uid = $uid.defined
+          ?? $uid.Int
```

Comment:
These should be `UInt`. see: https://github.com/MoarVM/MoarVM/pull/1725

Reply:
Nevermind, `-1.UInt` gives `-1` so that wouldn't matter

Reply:
The intent is achieved with `?? ($uid +& 0xffffffff)` and in fact safer.